Conference-Management-Tool-app-frontend/src/components/main/Main.jsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
-import {Redirect, Route, Switch} from 'react-router-dom';
+import { Redirect, Route, Switch } from 'react-router-dom';
 import ConferenceHolder from '../conference/ConferenceHolder';
 import DashBoardHolder from '../dashBoard/DashBoardHolder';
+import ReviewHolder from '../Review/ReviewHolder';
+import ResearchPaperHolder from '../Review/SubReviewer/ResearchPaperHolder';
+import WorkshopHolder from '../Review/SubReviewer/WorkshopHolder';
 
 /* functional component. */
 const Main = () => {
     return (
         <div style={{ height: '100%' }}>
             <Switch>
-                <Redirect exact from="/" to="/conferences"/>
+                <Redirect exact from="/" to="/conferences" />
                 {/* conferences component */}
                 <Route exact path="/conferences">
-                    <ConferenceHolder/>
+                    <ConferenceHolder />
                 </Route>
-                <Route path="/dashboard">
-                    <DashBoardHolder/>
```

Comment:
Why this bro?  if you change any code segment you have to check my components are working properly or not right....!

Reply:
@sachinthazoysa